schemas/common/autogeneratedResources.json

```diff
@@ -1672,6 +1672,12 @@
         {
           "$ref": "https://schema.management.azure.com/schemas/2020-08-20-preview/Microsoft.Communication.json#/resourceDefinitions/communicationServices"
         },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2020-10-01-preview/Microsoft.Compute.json#/resourceDefinitions/cloudServices"
+        },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2020-10-01-preview/Microsoft.Compute.json#/resourceDefinitions/cloudServices_updateDomains"
+        },
```

Comment:
This file is just for RPs that are currently onboarded, so these changes may get overwritten when the pipeline next runs. For now, please undo the changes to this file, and add the references to `schemas/2019-04-01/deploymentTemplate.json` instead.

Reply:
@anthony-c-martin confirming: per https://github.com/Azure/azure-resource-manager-schemas#submitting-a-pr it seems the references needs to be added in below 3 files; or is it only required for the `schemas/2019-04-01/deploymentTemplate.json`?
> Ensure that any new resource types that you are adding have been added to the following top-level template schemas:
schemas/2014-04-01-preview/deploymentTemplate.json
schemas/2015-01-01/deploymentTemplate.json
schemas/2019-04-01/deploymentTemplate.json

Reply:
Correct. The policy has changed w.r.t keeping the older root schemas up-to-date - we're only keeping 2019-04-01 updated going forwards. It's on my backlog to update the README.